billingbudgets/README.rst

```diff
@@ -0,0 +1,75 @@
+Python Client for Cloud Billing Budget API (`Alpha`_)
+=====================================================
```

Comment:
Needs a top-level summary description.

Reply:
[`billingbudgets.yaml`](https://github.com/googleapis/googleapis/blob/3fb0873b5f8a4cf5be4f848d20e5ccb2bdee1a67/google/cloud/billing/budgets/billingbudgets.yaml) is missing a summary line. I've asked the API team to add one.

Reply:
Do you want to wait on that change and update the PR with the same text?

Reply:
Yes I'll keep this open until I see the change to the protos